package.json

```diff
@@ -18,6 +18,7 @@
     "jscs": "^3.0.7",
     "semantic-release": "^6.3.6",
     "sinon": "^2.3.2",
-    "tap": "^10.3.2"
+    "tap": "^10.3.2",
+    "tmp": "0.0.33"
```

Comment:
This should be a prod dependency, not a dev dependency?

Reply:
- [X ]